shell/platform/darwin/ios/framework/Source/accessibility_bridge_test.mm

```diff
@@ -970,5 +970,6 @@ - (void)testAccessibilityMessageAfterDeletion {
   });
   latch.Wait();
   OCMVerify([messenger cleanupConnection:connection]);
+  [engine stopMocking];
```

Comment:
This might be superfluous now.  I don't think it hurts anything.  Once I got everything working I didn't try to cull any of the previous work I did.